uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -956,6 +956,30 @@ public function upgrade_1041() {
     return TRUE;
   }
 
+  /**
+   * Sets the default case type category to be Workflow.
+   *
+   * @return bool
+   */
+  public function upgrade_1042() {
+    $caseTypeWorkflowCategory = civicrm_api3('OptionValue', 'get', [
```

Comment:
I think it will be appropriate to rename the option value result `$caseTypeWorkflowCategory` to reflect option values 🤷‍♂️ .